contents/handbook/engineering/product-design.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Product Design, for Engineers
+sidebar: Handbook
+showTitle: true
+---
+
+It is up to you when to involve our product designers in your work.
+
+## v0.1 or v2?
+
+As the feature owner, you should make a choice if you're building a very basic first iteration of something, or if you're improving the experience.
+
+If you're improving an existing feature that is popular, you are probably creating v2.
+
+If you're shipping an entirely new feature (i.e. SQL for PostHog), then you should figure out if anyone cares, which usually means creating a v0.1, and releasing it behind a feature flag to some friendly users (pro tip: make friends by being support hero).
+
+Please _communicate_ to product design what your expectations are!
+
+### T-shirt sizes
```

Comment:
I don't get the reference to t-shirts, as I probably missed some meeting. Should I google this?

Reply:
It's a relative measurement for estimating the complexity or amount of work required to solve a problem. Extra-small, small, medium, large, etc. It's really just a loose way of saying this is a small problem or a big problem.

Reply:
todo: i'll reword above to remove the assumption people know this term